commerce_coordinator/settings/base.py

```diff
@@ -84,7 +84,9 @@ def root(*x):
 CORS_ALLOW_HEADERS = corsheaders_default_headers + (
     'use-jwt-cookie',
 )
-CORS_ORIGIN_WHITELIST = []
+CORS_ORIGIN_WHITELIST = [
+    'http://localhost:1996'
+]
 
```

Comment:
TODO: Follow up and see if this is the correct pattern/approach (I don't see this elsewhere)- for now want to get the endpoint working/tested (and with an empty set, requests were getting a Network Error)

Reply:
I believe this should go in `settings/local.py`; CORS is different in stage and prod because they have real domains.

Reply:
Ah of course- that makes sense! Moved and confirmed that frontend-app-ecommerce can still load the order history page via commerce-coordinator after the update